api/src/main/resources/workbench-api.yaml

```diff
@@ -9916,12 +9912,12 @@ definitions:
         type: integer
         format: int64
         description: |-
-          Prirmary key of the workspace table in Workrbench application database. Along with
+          Primary key of the workspace table in Workbench application database. Along with
           snapshot_timestamp, serves as a pseudo-primary key for this table.
       workspaceNamespace:
         type: string
         description: |-
-          The workspace "namespace", which corresponds to the Google Cloud Project ID associated with
+          The workspace "namespace", which corresponds to the Terra Billing Project ID associated with
```

Comment:
This is wrong: a Workspace Namespace *was* once equivalent to both a Google Project and a Terra Billing Project, but it is now only the later (since Terra's Project Per Workspace effort)